### PR TITLE
[15.0][MIG] stock_account_quantity_history_location: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT37317

Depends on:
  - [x] https://github.com/OCA/stock-logistics-reporting/pull/272

### CODE REVIEW

**Comment by ernestotejeda** [comment]
ping @chienandalu

**Review by chienandalu — APPROVED** [review]
Great :+1: . A possible (and easy) improvement (also in the main module) would be to add the location filter to the display name: https://github.com/odoo/odoo/blob/15.0/addons/stock_account/wizard/stock_quantity_history.py#L15

**Comment by ernestotejeda** [comment]
@DavidJForgeFlow could you review it, please?

**Review by DavidJForgeFlow — APPROVED** [review]
It may be good changing all Eficent mentions for ForgeFlow, but other than that it LGTM!

**Comment by ernestotejeda** [comment]
Changes done @chienandalu @DavidJForgeFlow

File: stock_account_quantity_history_location/wizards/stock_quantity_history.py [hunk 1]
```diff
@@ -35,4 +35,5 @@ def open_at_date(self):
                     ],
                 ]
             )
+            action["display_name"] += " ({})".format(self.location_id.complete_name)
```
Comment (chienandalu) [inline stock_account_quantity_history_location/wizards/stock_quantity_history.py]:
```suggestion
            action["display_name"] = f"{self.location_id.complete_name} - {action["display_name"]}"
```

**Review by chienandalu — CHANGES_REQUESTED** [review]
Watch out :warning:  you amended your last changes into the temporal commit `[DON'T MERGE] test-requirements.txt`

**Comment by ernestotejeda** [comment]
> Watch out warning you amended your last changes into the temporal commit `[DON'T MERGE] test-requirements.txt`

Right, I didn't realize it, now I fix it

**Comment by ernestotejeda** [comment]
changes done @chienandalu

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**